### PR TITLE
Updating DATS.json for compatibility with CONP portal requirements

### PR DESCRIPTION
**Pull request by emmetaobrien**
This PR updates the DATS.json file for this dataset for compatibility with https://github.com/CONP-PCNO/conp-dataset/pull/125 and https://github.com/CONP-PCNO/conp-dataset/pull/149, adding _formats_, _size_, _units_ and _access_ fields under _distribution_ and _contact_, _files_, _subjects_ and _CONP-status_ fields under _extraProperties_.

### CODE REVIEW

File: DATS.json [hunk 1]
```diff
@@ -36,6 +36,20 @@
         		"name": "Montréal"
         	}
     	],
+	"distributions": [
+ 		{
+ 			"formats": [
+ 				"mINC",
```
Comment (xlecours) [inline DATS.json]:
```suggestion
 				"minc",
```

Comment (xlecours) [inline DATS.json:42]:
```suggestion
 				"minc",
```

**Review by xlecours — CHANGES_REQUESTED** [review]

File: DATS.json (hunk 1 above)
Comment (xlecours) [reply DATS.json:42]: @emmetaobrien

Comment (emmetaobrien) [reply DATS.json:42]: OK, done

**Pull request merged**